README.md

```diff
@@ -1,60 +1,63 @@
+PSyclone is a source-to-source Fortran compiler designed to programmatically
+optimise and parallelise HPC applications using user-provided transformation
+scripts. These scripts offer a separation of concerns between the science code
+and the performance portability implementation choices, allowing to experiment
+and develop with each side independenly. Additonally, PSyclone supports the
+development of kernel-based Fortran-embedded DSLs following the PSyKAl model
+developed in the [GungHo project](https://www.metoffice.gov.uk/research/foundation/dynamics/next-generation).
 
-# Try it on Binder #
+PSyclone is currently used to support the
+[LFRic PSyKAl mixed finete elements DSL](https://www.metoffice.gov.uk/research/modelling-systems/lfric/)
+for the UK MetOffice's next generation modelling system and the
+[GOcean finite difference prototype DSL](https://gtr.ukri.org/projects?ref=NE%2FL01209X%2F1).
+It is also used to insert GPU offloading directives to existing
+directly-addressed MPI applications such as the [NEMO ocean model](https://www.nemo-ocean.eu/).
 
-Some of the examples are available as Jupyter notebooks. These may
-be launched using Binder from the links below. (Note that the first time
-this is done, Binder has to construct a Container and install the necessary
-software. This can  take several minutes. You can track its progress
-by clicking the 'show' link next to the 'Build logs' heading.)
+For more detailed information see the [psyclone.pdf](psyclone.pdf) in this directory
+or the [PSyclone User's Guide](http://psyclone.readthedocs.io).
 
- * [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/master?filepath=examples%2Fnemo%2Feg4%2Fcopy_stencil.ipynb) Uses PSyclone's NEMO API to process some simple Fortran code, display the resulting PSyIR and then re-generate Fortran.
-
- * [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/master?filepath=examples%2Fgocean%2Feg1%2Fopenmp.ipynb) Uses PSyclone's GOcean API to process example code that conforms to the PSyKAl separation of concerns. Transformations are applied in order to fuse various loops before parallelising the result with OpenMP.
- 
- * [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/master?filepath=examples%2Fgocean%2Feg1%2Fdag.ipynb) demonstrates the generation of a DAG for the PSy layer of the previous example.
+# Installation #
 
-# Tutorial #
+You can install the latest release of psyclone from PyPI by using:
 
-The PSyclone tutorial may be found in the tutorial directory. Since part of
-this takes the form of a series of Jupyter notebooks, it too may be launched on
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/master?filepath=tutorial%2Fnotebooks%2Fintroduction.ipynb).
+    $ pip install psyclone
 
-# Installation #
+or, if you do not have root access or want an isolated installation:
```

Comment:
I think `pip` does the right thing these days if you don't have root access - it just installs it under ~/.local.

Reply:
Oh cool, removed this part, but I still think it's useful to mention the virtualenv steps, as it's what most users will do.